### DISCUSSION TITLE
Error - All declarations of 'Commands' must have identical type parameters.

### DISCUSSION THREAD

**Discussion by euvl** [ueberdosis/tiptap#1432] [General Questions] [answered]
Hi guys, i might need some help with this typescript error:

```
interface Commands<ReturnType>

All declarations of 'Commands' must have identical type parameters.
```

All my interfaces for custom elements look like,

```
declare module "@tiptap/core" {
  interface Commands<ReturnType> {
    remark: {
      setRemark: (attributes?: { remark: string }) => ReturnType;
      unsetRemark: () => ReturnType;
    };
  }
}
```

started happening after I've imported list item + ordered list + bullet list  (🤷 ). I've deleted those but still getting this error. Any ideas what this might be?

NOTE:

When it started happening - it happens within every file, I presume it's because we expected module "@tiptap/core"  => interface Commands<ReturnType>

**Comment by philippkuehn** [comment, marked as answer]
Hey this is a breaking change since the latest release. Please try updating all tiptap packages. This should help.

**Reply by euvl** [reply]
ah got you! Thanks
